src/components/ui/Flex.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import classnames from 'classnames';
+
+const Flex = ({ children, inline, spaceBetween, alignCenter, className }) =>
+  <div className={classnames('flex', {
+    'flex--spaceBetween': spaceBetween,
+    'flex--alignCenter': alignCenter,
```

Comment:
What about using props named `align` and `justify` for instance ? This way user could set `align='center/start/end'` etc

Reply:
I wanted something more declarative and easier for calling code, with just bool props and no 1-to-1 mapping to real flex rules, otherwise there is not much added value compared to just using classes or inline styles.
This is also the approach taken by the design system team (but I can't link to the code because… you know :roll_eyes:). But the prop names are not the same, I should have taken them.
So I guess we can discuss the best action:
 - keeping this approach, but changing the props to match the DS component
 - use your approach, with classic key=value props

Reply:
If we keep the current approach, the thing is nothing prevent the user to set contradictory props like settings both `alignCenter` and `alignStart` for instance.
By offering an API which restrict possibilities with prop types, we can ensure a predictable behavior and keep a minimal amount of props.

Reference:
https://rebassjs.org/reflexbox/#flexbox-props

As for current Design System implementation, it seems they chose to go with a higher level api, like props `center` setting both `align-items: center` and justify-content: `center` I guess. I think this could be limiting for some case, but could be shorthands for other props.

Reply:
I used props matching flex rules like you proposed, limiting to the ones we really need for now.